hooks/playbooks/run_tofu.yml

```diff
@@ -19,6 +19,21 @@
   gather_facts: true
   run_once: true
   tasks:
+    - name: Create openstack config dir
+      ansible.builtin.file:
+        path: /home/zuul/.config/openstack/
```

Comment:
Can we use the ansible_user_dir here?

Reply:
depends where the role is called - nested ansible, delegate_to, or... ?